packages/webview_flutter/webview_flutter_wkwebview/example/lib/main.dart

```diff
@@ -272,6 +318,20 @@ class _SampleMenu extends StatelessWidget {
       children: cookieWidgets.toList(),
     );
   }
+
+  static Future<String> _prepareLocalFile() async {
+    final String tmpDir = (await getTemporaryDirectory()).path;
+    File indexFile = File('$tmpDir/www/index.html');
+
+    if (await indexFile.exists()) {
+      return indexFile.path;
```

Comment:
You are right. I added this check with the idea that if the file exists we don't have to write it again (in case the user presses the button twice in the same session). However I didn't think about the fact that the file will be persisted over multiple sessions and developers might want to change the HTML contents.